frontend/src/components/pending-sightings-page/PendingSightingCard.scss

```
@@ -0,0 +1,7 @@
+.sighting-card {
+  margin: 10px;
+}
+
+.sighting-card--error {
```

Apparently, the two dashes here should be underscores 🤷

```suggestion
.sighting-card__error {
```